Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Provided resources: mem_mb=100000, mem_mib=95368, disk_mb=1000, disk_mib=954, nvidia_gpu=1
Select jobs to execute...

[Tue Jun 18 00:20:52 2024]
rule run_experiment:
    input: zoutput/benchmarks/CAMELYON_center0_erm_resnet_different_batchsizes_lre3/hyperparameters.csv
    output: zoutput/benchmarks/CAMELYON_center0_erm_resnet_different_batchsizes_lre3/rule_results/4.csv
    jobid: 0
    reason: Forced execution
    wildcards: index=4
    resources: mem_mb=100000, mem_mib=95368, disk_mb=1000, disk_mib=954, tmpdir=/tmp, partition=gpu_p, qos=gpu_normal, nvidia_gpu=1

Failed to import backpack: cannot import name '_grad_input_padding' from 'torch.nn.grad' (/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/torch/nn/grad.py)
[Tue Jun 18 00:21:05 2024]
Error in rule run_experiment:
    jobid: 0
    input: zoutput/benchmarks/CAMELYON_center0_erm_resnet_different_batchsizes_lre3/hyperparameters.csv
    output: zoutput/benchmarks/CAMELYON_center0_erm_resnet_different_batchsizes_lre3/rule_results/4.csv

RuleException:
KeyError in file /ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/benchmark.smk, line 161:
4
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/benchmark.smk", line 161, in __rule_run_experiment
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/run_experiment.py", line 80, in run_experiment
  File "/ictstr01/groups/aih/sina.wendrich/MA_code/MA_thesis/DomainLab/domainlab/exp_protocol/run_experiment.py", line 28, in load_parameters
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1073, in __getitem__
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1312, in _getitem_axis
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexing.py", line 1260, in _get_label
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/generic.py", line 4056, in xs
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3804, in get_loc
  File "/home/aih/sina.wendrich/miniconda3/envs/domainlab_py310/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
